gsmphys/rad_initialize.f

```diff
@@ -136,12 +137,13 @@ subroutine rad_initialize                                         &
 
       ictmflg= ictm                     ! data ic time/date control flag
       ico2flg= ico2                     ! co2 data source control flag
+      co2_scaling= fco2_scaling         ! co2 level scaling
```

Comment:
May want to put a comment here that you are re-setting the value of co2_scaling in physparam.